drive/client.py

```diff
@@ -243,7 +243,7 @@ def list_files(self,
                    name_contains: Optional[str] = None,
                    mimetype: Optional[str] = None,
                    parents_in: Optional[str] = None,
-                   n=100):
+                   n: int = 100) -> List[File]:
```

Comment:
`: int` is not needed because `100` is already an `int`. `List[File]` is redundant with the `return cast(...)`

Reply:
See above: https://github.com/bfontaine/drive/pull/13#discussion_r2015341970